### PR TITLE
testkit: Enable browser tests log on terminal

### PR DESCRIPTION
**Pull request by bigmontz**
Introduce a Console message in the protocol between the LocalController on Browser and the RemoteController in Node for enabling the logs and other console messages to be printed in terminal and be available in artifacts.

This feature should help debugging the errors in the testing pipeline.

### CODE REVIEW

File: packages/testkit-backend/src/console.remote.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  handleConsole: (message) => {
+    if (message.response.name === 'Console') {
+      const { method, args } = message.response.data
+      args[0] = typeof args[0] === 'string' ? `[RemoteConsole] ${args[0]}` : args[0]
+      console[method].apply(console, args)
+      return true
+    }
+    return false
+  },
+  uninstall: () => {
```
Comment (StephenCathcart) [inline packages/testkit-backend/src/console.remote.js:27]: More of a question - do we need/use this uninstall function or is this just for future-proofing?

**Review by StephenCathcart — APPROVED** [review]
LGTM! 🚀

File: packages/testkit-backend/src/console.remote.js (hunk 1 above)
Comment (bigmontz) [reply packages/testkit-backend/src/console.remote.js:27]: We don't need, this is only implemented for future-proofing and for keeping the symmetry.

Comment (StephenCathcart) [reply packages/testkit-backend/src/console.remote.js:27]: Brilliant sounds good to me, just checking in case it was missing somewhere 👍

**Pull request merged**